{"name":"negativeLogInTest","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.lang.RuntimeException: File src\\main\\resouces\\DataProvider\\NegativeLoginTestnegativeLogInTest.csv was not found.\n[Ljava.lang.StackTraceElement;@7a6d7e92","trace":"java.lang.RuntimeException: java.lang.RuntimeException: File src\\main\\resouces\\DataProvider\\NegativeLoginTestnegativeLogInTest.csv was not found.\n[Ljava.lang.StackTraceElement;@7a6d7e92\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:49)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:131)\r\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:706)\r\n\tat org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:49)\r\n\tat org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:37)\r\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:924)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:648)\r\n\tat org.testng.TestRunner.run(TestRunner.java:505)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:455)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:364)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1137)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1049)\r\n\tat org.testng.TestNG.run(TestNG.java:1017)\r\n\tat org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)\r\n\tat org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)\r\nCaused by: java.lang.RuntimeException: File src\\main\\resouces\\DataProvider\\NegativeLoginTestnegativeLogInTest.csv was not found.\n[Ljava.lang.StackTraceElement;@7a6d7e92\r\n\tat Utils.CsvDataProviders.csvReader(CsvDataProviders.java:37)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:45)\r\n\t... 21 more\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1612338891717,"stop":1612338891721,"uuid":"45ea5592-2f04-44ea-a442-9032a333a5ad","historyId":"70bf6821bf8e7d79f24f47e531ddc236","fullName":"TestCases.NegativeLoginTest.negativeLogInTest","labels":[{"name":"package","value":"TestCases.NegativeLoginTest"},{"name":"testClass","value":"TestCases.NegativeLoginTest"},{"name":"testMethod","value":"negativeLogInTest"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"AdvanceSeleniumTraining"},{"name":"subSuite","value":"TestCases.NegativeLoginTest"},{"name":"host","value":"s-chimbay-pc"},{"name":"thread","value":"34372@s-chimbay-pc.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[]}